website/content/docs/agent/telemetry.mdx

```diff
@@ -89,6 +89,23 @@ These are some metrics emitted that can help you understand the health of your c
 
 **What to look for:** For a healthy cluster, you're looking for a `lastContact` lower than 200ms, `leader` > 0 and `candidate` == 0. Deviations from this might indicate flapping leadership.
 
+### Certificate Authority Expiration
+
+| Metric Name                            | Description                                                             | Unit    | Type  |
+| :------------------------- | :---------------------------------------------------------------------------------- | :------ | :---- |
+| `consul.mesh.active-root-ca.expiry`    | The number of seconds until the root CA expires, updated every hour.    | seconds | gauge |
+| `consul.mesh.active-signing-ca.expiry` | The number of seconds until the signing CA expires, updated every hour. | seconds | gauge |
+
+** Why they're important:** Consul Mesh requires a CA to sign all certificates
+used to connect the mesh and the mesh network ceases to work if they expire and
+become invalid. The Root is particularly important to monitor as Consul does
+not automatically rotate it.
+
+** What to look for:** The Root CA should be monitored for an approaching
+expiration, to indicate it is time to manually rotate the CA or to verify your
+automated system is working. The signing (intermediate) certificate should be
+rotated automatically by Consul but it is good to monitor it in case of issues.
```

Comment:
Optional clarification to consider
```suggestion
rotated automatically by Consul, but should be monitored in case of rotation isn't working;
in this scenario, check the server agent logs for messages related to the CA system.
```